src/sirhelpers.c

```diff
@@ -93,6 +93,9 @@ bool _sir_validfd(int fd) {
 }
 
 /** Validates a sir_update_config_data structure. */
+# if defined(__clang__) /* only Clang has implicit-conversion; GCC BZ#87454 */
+SANITIZE_SUPPRESS("implicit-conversion")
```

Comment:
I will need to know what the implicit conversion warnings in every place we are putting this are